Automattic/sensei
# Fix custom styles for emails in preview and in sent messages

Resolves #7459 

Some styling that comes by default with WordPress couldn't be applied to emails. 

The main restriction comes from Gmail that removes unsupported styling from emails in their web-app. For this reason, we disable some settings in this PR.

We address the issue with misstyling of elements due to lack or conflict of styles.

## Proposed Changes
* Load global styles for inline CSS processor.
* Disable default color palettes.
* Disable alignment for buttons and images.
* Remove some default from editor settings that affect the appearance of the email in the editor only.

## Testing Instructions

For testing real messages, use Gmail webapp as it has more restrictive rules for CSS comparing to desktop applications.

1. Go to Sensei LMS > Settings > Emails.
2. Enable the Course Completed email and then edit it.
3. Change colors for buttons, paragraphs.
4. Change text alignment.
5. Add an image.
6. Make sure you don't have controls to change alignment of the image or buttons.
7. Save and preview. Make sure you see your changes in the preview.
8. Take a look at the button and make sure it has the same padding as in the editor.
9. Complete the course as a student.
10. Check your mailbox and make sure that the Course Completed email reflects your changes.

## Screenshots

### Real email 
(You can see that Sensei logo and the image in the body are not displayed — that's because they refer to my local website.)

#### Gmail web-app
![CleanShot 2024-06-02 at 15 29 49@2x](https://github.com/Automattic/sensei/assets/329356/b4efe93b-43ff-4d86-b89a-e61b24c44c9c)

#### Mail App
![CleanShot 2024-06-02 at 15 30 24@2x](https://github.com/Automattic/sensei/assets/329356/534603b1-47a5-4f37-a77d-364a839493ad)

### Editor
![CleanShot 2024-06-02 at 15 27 31@2x](https://github.com/Automattic/sensei/assets/329356/8a1eb008-95ad-49c7-8e76-ffb871d0ec20)

### Preview
![CleanShot 2024-06-02 at 15 27 44@2x](https://github.com/Automattic/sensei/assets/329356/e4b20f53-3313-4495-895a-fc619ea5eb23)

## Pre-Merge Checklist
